arch/arm64/boot/dts/marvell/accton-as4560-52p.dts

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Device Tree For Accton as4560-52p.
+ *
+ * Copyright (C) 2021 Marvell
+ *
+ */
+
+#include "accton-as4560-52p.dtsi"
+
+/ {
+	model = "Marvell AC5X RD board";
+};
```

Comment:
I would have expected a `chosen` node here like in https://github.com/dentproject/linux/blob/dent-linux-5.15.105/arch/arm64/boot/dts/marvell/delta-tn48m-dn.dts#L52 that describes the default serial console. Or is there none?